app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    main_file = parts[1]
+    for_copy_file = parts[2]
```

Comment:
Consider using unpacking for improved readability and to avoid repeating variable assignments